docs/devel/development.md

```diff
@@ -325,9 +325,10 @@ Three basic commands let you run unit, integration and/or e2e tests:
 
 ```sh
 cd kubernetes
-make test  # Run unit tests
+make test # Run every unit test
+make test WHAT=pkg/util/cache GOFLAGS=-v # Run tests of a package verbosely
 make test-integration  # Run integration tests, requires etcd
-go run hack/e2e.go -v --build --up --test --down  # Run e2e tests
+make test-e2e # Run e2e tests
```

Comment:
The Makefile target doesn't have ginkgo args (I'm assuming its because the ginkgo args are space separated and Makefile makes it a bit harder), but I added a link to the E2E testing guide in case people want to run customized tests.